eap74-rhel8-byos-multivm/src/main/scripts/create-ds.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+log() {
```

Comment:
Can you please remind me once again why we don't need the `set -Eeuo pipefail` here?